src/controllers/employer.ts

```diff
@@ -119,20 +119,6 @@ export const updateEmployer = async (
   }
 }
 
-// getting employer based on matched credentials
-export const getEmployer = async (
-  req: Request,
-  res: Response,
-  next: NextFunction
-) => {
-  try {
-    const user = await Employer.find({ relations: ['credentials'] })
-    res.json(user)
-  } catch (error) {
-    console.log(error)
-  }
-}
-
 export const createJobPost = async (
```

Comment:
Update this controller to make it create a job post with skills also. Currently it is only saving basic info of a post.
The response body should have a list of skills ids, then you query to the skills table to find those skills, put the skills found in the **requiredSkills** column and save the job post. This will tell **typeorm** to save their relation